core/new-gui/src/app/workspace/component/navigation/navigation.component.html

```diff
@@ -89,6 +89,9 @@
         <button (click)="onClickRestoreZoomOffsetDefault()" nz-button title="reset zoom">
           <i nz-icon nzTheme="outline" nzType="fullscreen"></i>
         </button>
+        <button (click)="onClickAddCommentBox()" nz-button title="add a comment">
```

Comment:
does the annotation work if user system is disabled? (in dev mode)

Reply:
Yes, I think I have it tested.